tests/test_2024_04_23.py

```diff
@@ -0,0 +1,132 @@
+"""
+Tests for voting 23/04/2024
+
+"""
+
+from brownie import Contract, web3, chain  # type: ignore
+from brownie import reverts, accounts, interface
+from scripts.vote_2024_04_23 import start_vote
+from utils.voting import find_metadata_by_vote_id
+from utils.ipfs import get_lido_vote_cid_from_str
+from utils.config import (
+    contracts,
+    network_name,
+
+    GATE_SEAL_PAUSE_DURATION, 
+    GATE_SEAL_COMMITTEE,
+)
+from utils.test.tx_tracing_helpers import *
+from utils.test.event_validators.permission import (
+    validate_grant_role_event,
+    validate_revoke_role_event
+)
+
+PAUSE_ROLE = "0x139c2898040ef16910dc9f44dc697df79363da767d8bc92f2e310312b816e46d" # web3.keccak(text="PAUSE_ROLE")
+old_gate_seal = "0x1ad5cb2955940f998081c1ef5f5f00875431aa90"
+new_gate_seal = "0x79243345eDbe01A7E42EDfF5900156700d22611c"
+
+def _check_role(contract: Contract, role: str, holder: str):
+    role_bytes = web3.keccak(text=role).hex()
+    assert contract.getRoleMemberCount(role_bytes) == 1, f"Role {role} on {contract} should have exactly one holder"
+    assert contract.getRoleMember(role_bytes, 0) == holder, f"Role {role} holder on {contract} should be {holder}"
+
+def _check_no_role(contract: Contract, role: str, holder: str):
+    role_bytes = web3.keccak(text=role).hex()
+    assert contract.getRoleMemberCount(role_bytes) == 1, f"Role {role} on {contract} should have exactly one holder"
```

Comment:
it was my bad, better to remove this check 'contract.getRoleMemberCount(role_bytes) == 1,'

Reply:
agreed to delete